osu.Server.DifficultyCalculator/Program.cs

```diff
@@ -13,6 +13,7 @@ namespace osu.Server.DifficultyCalculator
     [Subcommand("files", typeof(FilesCommand))]
     [Subcommand("beatmaps", typeof(BeatmapsCommand))]
     [Subcommand("since", typeof(SinceCommand))]
+    [Subcommand("beatmapsstring", typeof(BeatmapsStringCommand))]
```

Comment:
need to update all these subcommands at some point to not specify the command name (is obsoleted).

Reply:
just the type?

Reply:
it enough for it to work, yeah.